app/views/layouts/mailer.html.erb

```diff
@@ -95,20 +95,14 @@
                             <p><%= yield(:footer_note) %></p>
                           <% end %>
 
-                          <p>
-                            If you've received this message in error, please <a href="<%= root_url %>support" style="color:#ffffff; text-decoration:underline">contact Support</a>.
-                          </p>
-
-                          <p>
-                            The Archive of Our Own is a fan-run and fan-supported archive that relies on <a href="<%= donate_url %>" style="color:#ffffff; text-decoration:underline">your donations</a>.
-                          </p>
+                          <%= t 'mailer.general.footer.general.html', support_url: root_url+"support", donate_url: donate_url %>
 
                           <% if content_for?(:sent_at) %>
-                            <p>Sent at: <%= yield(:sent_at) %>.</p>
+                            <p><%= t 'mailer.general.footer.sentat'%><%= yield(:sent_at) %>.</p>
```

Comment:
It looks as if the localised strings are pushed up against the date/time, so in English you'd get "Sent at2015-03-15T12:35:00" (or whatever the date/time format is). I also don't know how well this arrangement would work for, say, Arabic - or any language that needs to add something after the date (hypothetically). You should probably incorporate the `:sent_at` using interpolation instead.